pages/style-test.js

```diff
@@ -0,0 +1,31 @@
+import { styled } from '@material-ui/core/styles';
+import Button from '../components/Button';
```

Comment:
We import our `Button` component. Then we can use it as is, or apply additional styles using `styled-components`.